src/environments/environment.prod.ts

```diff
@@ -21,7 +21,7 @@ export const environment = {
       idamLoginUrl: 'https://hmcts-access.service.gov.uk',
       idamWeb: 'https://hmcts-access.service.gov.uk',
       indexUrl: '/',
-      oauthCallbackUrl: 'oauth2/callback'
+      oauthCallbackUrl: '/oauth2/callback'
```

Comment:
Just be careful here, I'm guessing it needs an extra slash now.

Reply:
I am not sure about the extra slash. Let's keep it for now. We had to remove it when were doing the openId connect @ficklephil